src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractAccessControlNameCheck.java

```diff
@@ -116,6 +116,7 @@ private static boolean isPublic(DetailAST modifiers) {
      * Setter to control whether we should apply the check to public members.
      *
      * @param applyTo new value of the property.
+     * @since 5.0
```

Comment:
We are missing tags for this release, and 4.4 and 5.2 through 5.7 tags prefixed with `checkstyle-` are wrong.

Move from snapshot 4.4 to 5.0 was bcbea917043b2657aa39c735ec70494c9b1185c1 . Tagged 4.4 release was 3fb7ae8ee60c30f1ad4a5b4b2ad0325075fef6ac .
5.0 release was f31928524128c0909347456ef33105e63ee59824 .
5.1 release was 49d74e731fb9d8e6d3e82123f6cb0f8e4f9b10b4 .

Reply:
Class existed in f31928524128c0909347456ef33105e63ee59824 @ https://github.com/checkstyle/checkstyle/blob/f31928524128c0909347456ef33105e63ee59824/src/checkstyle/com/puppycrawl/tools/checkstyle/checks/naming/AbstractAccessControlNameCheck.java
Class did not in 3fb7ae8ee60c30f1ad4a5b4b2ad0325075fef6ac @ https://github.com/checkstyle/checkstyle/tree/3fb7ae8ee60c30f1ad4a5b4b2ad0325075fef6ac/src/checkstyle/com/puppycrawl/tools/checkstyle/checks/naming

Reply:
all methods appear at Feb 21, 2008  https://github.com/checkstyle/checkstyle/commit/f6b0b3cd7f1587e97bbf188e8b285935e376f5ef#diff-1a94a2039f30e0d7150a18f2f520d1c781a616de7b170ed569e662c501ecdeb7R110

https://sourceforge.net/projects/checkstyle/files/OldFiles/
![image](https://github.com/checkstyle/checkstyle/assets/812984/121be21b-5f49-4e8e-bd99-cb50d51610f9)

it is 5.0